parse/EnumPythonParser.cpp

```diff
@@ -84,5 +84,16 @@ void RegisterGlobalsEnums(boost::python::dict& globals) {
         globals[op.first] = enum_wrapper<PlanetType>(op.second);
     }
 
+    for (const auto& op : std::initializer_list<std::pair<const char*, UnlockableItemType>>{
+            {"Building",   UnlockableItemType::UIT_BUILDING},
+            {"ShipPart",   UnlockableItemType::UIT_SHIP_PART},
+            {"ShipHull",   UnlockableItemType::UIT_SHIP_HULL},
+            {"ShipDesign", UnlockableItemType::UIT_SHIP_DESIGN},
+            {"UnlockTech", UnlockableItemType::UIT_TECH},
```

Comment:
having `Unlock` in the token for just techs is awkward

Reply:
But just `Tech` token already used to declare techs itself.

Reply:
makes sense, but why are all the other tokens not also already used the same way?

Reply:
Sounds reasonable for future conversion.

Reply:
Changed it everywhere.